SUPPORT.md

```diff
@@ -0,0 +1,5 @@
+# Ark Support
```

Comment:
Can you add a link to this in the main README? perhaps under "Troubleshooting" or "Contributing"? (whichever you think is best)? 

Also if we are doing this for Sonobuoy, keep them consistent :) thanks!